handlers/main.yml

```diff
@@ -6,3 +6,7 @@
 - name: restart celery
   supervisorctl: name=celery state=present
   supervisorctl: name=celery state=restarted
+
+- name: restart celery events
```

Comment:
If we're always restarting both celery processes, we could just combine them into a single notify.